concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1798,6 +1797,27 @@ jobs:
 
                   ../paas-cf/concourse/scripts/cf_push_on_git_change.sh graphite-nozzle
 
+        - task: run-bosh-cleanup
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/bosh-cli
+                tag: 895cf6752c8ec64af05a3a735186b90acd3db65a
+            inputs:
+              - name: paas-cf
+              - name: bosh-secrets
+            run:
+              path: sh
+              args:
+                - -e
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/bosh_login.sh {{bosh_fqdn}} bosh-secrets/bosh-secrets.yml
+
+                  bosh cleanup --all
```

Comment:
This addition looks to be erroneous (this is a duplicate of the existing job in this pipeline). I've raised #1062 to remove it.